content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,154 @@
+
+### Configuration
+
+The Topology Manager currently:
+* Works on nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/)
+* Works on Pods in the `Guaranteed` QOS Class {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}. 
+If these conditions are met, Topology Manager will align CPU and device requests.
+
+Topology Manager supports two allocation policies. You can set a policy via a Kubelet flag, `--topology-manager-policy`.
+There are three supported policies:
+
+* `none` (default)
+* `best-effort`
+* `restricted`
+* `single-numa-node`
+
+### none policy {#policy-none}
+
+This is the default policy and does not perform any topology alignment.
+
+### best-effort policy {#policy-best-effort}
+
+For each container in a Guaranteed Pod, kubelet, with `best-effort` topology 
+management policy, calls each Hint Provider to discover their resource availability.
+Using this information, the Topology Manager stores the 
+optimal NUMA Node affinity for that container. If the affinity is not preferred, 
+Topology Manager will store this and admit the pod to the node.
+
+The *Hint Providers* can then use this information when making the 
+resource allocation decision.
+
+### restricted policy {#policy-restricted}
+
+For each container in a Guaranteed Pod, kubelet, with `restricted` topology 
+management policy, calls each Hint Provider to discover their resource availability.
+Using this information, the Topology Manager stores the 
+optimal NUMA Node affinity for that container. If the affinity is not preferred, 
+Topology Manager will reject this pod from the node. This will result in a pod in the `Pending` state.
+
+The *Hint Providers* can then use this information when making the 
+resource allocation decision.
+
+### single-numa-node policy {#policy-single-numa-node}
+
+For each container in a Guaranteed Pod, kubelet, with `single-numa-node` topology 
+management policy, calls each Hint Provider to discover their resource availability.
+Using this information, the Topology Manager determines if a single NUMA Node affinity is possible.
+If it is Topology Manager will store this and the *Hint Providers* can then use this information when making the 
+resource allocation decision.
+If, however, this is not possible the Topology Manager will reject the pod from the node. This will result in a pod in the `Pending` state.
+
+
+### Example Usage
+
+Consider the containers in the following pod specs:
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```
+
+This pod runs in the `BestEffort` QoS class because no resource `requests` or
+`limits` are specified.
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+      requests:

```

Comment:
You can remove the requests if you want to simplify this. If just limits is set, it assumes `requests == limits`.